packages/Utils/src/DTK_DBC.hpp

```diff
@@ -63,7 +63,9 @@ class DataTransferKitException : public std::logic_error
     }
 
     //! Destructor.
-    virtual ~DataTransferKitException() throw() { /* ... */}
+    virtual ~DataTransferKitException() throw()
+    { /* ... */
+    }
```

Comment:
How about that one :/

Reply:
yeah that sucks but it probably always wants to separate the '{}'

Reply:
Actually if you look at `DTK_DBC.hpp`, you will see that the two constructors for `DataTransferKitException` do this as well. This destructor should probably be `= default;` anyway.